drop-analytics/src/file_impl.rs

```diff
@@ -78,7 +79,13 @@ impl FileImpl {
         };
         events.push(event);
 
-        File::create(&self.event_path)?.write_all(serde_json::to_string(&events)?.as_bytes())?;
+        let payload = serde_json::to_string(&events)?;
+
+        // create unique temp path
+        let temp_path = format!("{}.tmp.{}", self.event_path, uuid::Uuid::new_v4());
+
+        File::create(&temp_path)?.write_all(payload.as_bytes())?;
```

Comment:
```suggestion
        std::fs::write(&temp_path, payload.as_bytes())?;
```

Reply:
It's a minor thing but it still has not changed. If you want to ignore it, go ahead

Reply:
I wonder what difference does it make. It should be in faster, in theory, without creating an instance but still wonder

Reply:
updated it

Reply:
Probably there is no difference. Just more direct API